scaleway/resource_baremetal_server_beta.go

```diff
@@ -0,0 +1,220 @@
+package scaleway
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	baremetal "github.com/scaleway/scaleway-sdk-go/api/baremetal/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func resourceScalewayBaremetalServer() *schema.Resource {
```

Comment:
`resourceScalewayBaremetalServerBeta`

same for all function and test. When we release the stable version we will need to maintain both versions at the same time.